### DISCUSSION TITLE
measure traffic volume of an entire profil (e.g. GOS)

### DISCUSSION THREAD

**Discussion by sudo-joe** [celzero/rethink-app#1719] [General]
Hi,

1.
Would find it cool if the entire traffic volume of a profile. e.g. the one of GrapheneOS could be "measured"

2.
In addition it would be great if the timeframe of this traffic measurement couold be defined, e.g. from 08.00 till 08.45

Thank's!

**Comment by ignoramous** [comment]
> Would find it cool if the entire traffic volume of a profile. e.g. the one of GrapheneOS could be "measured"

The *Stats* screen already shows per-app bandwidth consumed. Is that not enough? What do you mean when you say "profile"?

> addition it would be great if the timeframe of this traffic measurement couold be defined, e.g. from 08.00 till 

The *Stats* screen today has 1hr, 1day, 1week slots pre-defined. Not enough?

**Comment by sudo-joe** [comment]
>The Stats screen already shows per-app bandwidth consumed. Is that not enough? What do you mean when you say "profile"?

With profile I do mean a user profile of GrapheneOS.

>The Stats screen today has 1hr, 1day, 1week slots pre-defined. Not enough?

Yes but it would be cool to be able to drill down the traffic the app is generating and to know what traffic Google-services do generate  (Google Play Services + Google Play Store + Google Services Framework)
